tests/ha/test_ha_pod_restart.py

```diff
@@ -1818,14 +1820,16 @@ def test_copy_object_during_pod_restart(self):
                 'object_name': self.object_name, 'bkt_obj_dict': bkt_obj_dict1, 'output': output,
                 'file_path': self.multipart_obj_path, 'background': True, 'bkt_op': False,
                 'put_etag': put_etag}
-        prc = Process(target=self.ha_obj.create_bucket_copy_obj, kwargs=args)
-        prc.start()
-        LOGGER.info("Step 6: Successfully started background process")
-
+        thread = threading.Thread(target=self.ha_obj.create_bucket_copy_obj, args=(event,),
+                                  kwargs=args)
+        thread.daemon = True  # Daemonize thread
+        thread.start()
+        LOGGER.info("Step 6: Successfully started background process for copy object")
         # Delay added to sync this operation with background thread to achieve expected scenario
-        time.sleep(HA_CFG["common_params"]["30sec_delay"])
+        time.sleep(HA_CFG["common_params"]["90sec_delay"])
```

Comment:
why increased delay?

Reply:
So in HA_CFG["copy_obj_data"]["bkt_multi"] we will have to create 10 buckets in background which will take time. So to sync foreground and background process, we have to increase this delay.

Reply:
Can we rather check then that all buckets are created and then start the foreground process instead of this constant value?